### PR TITLE
[uefi] add mouse pointer supported. the cursor render by using minipng.

### PR DESCRIPTION
**Pull request by sky5454**
originally in https://github.com/slint-ui/slint/pull/5034

### CODE REVIEW

File: examples/uefi-demo/README.md [hunk 1]
```diff
@@ -5,23 +5,28 @@
 This example demonstrates Slint in a UEFI environment.
 
 ![Screenshot](https://user-images.githubusercontent.com/1486/231705364-8c490e25-48cf-4626-a34b-2bf7239c1245.jpg)
+![img.png](https://github.com/slint-ui/slint/assets/12370628/d329f6ee-184f-4c62-8b36-e32123211685)
+the red rectangle is mouse. Here's how it works:
```
Comment (ogoffart) [inline examples/uefi-demo/README.md:9]: It is no longer a red rectangle. I think these two lines can be removed.

**Review by ogoffart — APPROVED** [review]
Thanks for the cleanups.

**Comment by sky5454** [comment]
Any other problem? merge it if nothing need to fix.

And the next plan, I would add uefi to Slint-UI 's `internal/backends/uefi`

**Comment by ogoffart** [comment]
One last thing I need before merging is the license for `cursor.png`. Do you know where this file comes from?

**Comment by sky5454** [comment]
> One last thing I need before merging is the license for `cursor.png`. Do you know where this file comes from?

from https://icons8.com/icon/11945/cursor

**Comment by ogoffart** [comment]
Merged!

Thanks a lot for your contribution!

**Pull request merged**